src/config/subgraph.ts

```diff
@@ -22,14 +22,15 @@ const SUBGRAPH_URLS = {
     stats: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-avalanche-stats/api",
     referrals: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/gmx-avalanche-referrals/api",
     syntheticsStats: "https://subgraph.satsuma-prod.com/3b2ced13c8d9/gmx/synthetics-avalanche-stats/api",
-    leaderboard: "https://squid.subsquid.io/gmx-synthetics-avalanche/graphql",
+    subsquid: "https://gmx.squids.live/gmx-synthetics-avalanche/v/v14/graphql",
```

Comment:
just checking, should move squid to prod or we need to deploy UI with fixed version first?

Reply:
just noticed hostname has been changed,
yes, I think it's fine to deploy subsquid to prod and not touch urls here (unless update hostname, just in case)